rosdep/python.yaml

```diff
@@ -5543,6 +5543,22 @@ rpy2:
   debian: [python-rpy2]
   gentoo: [=dev-python/rpy-2*]
   ubuntu: [python-rpy2]
+selectors2-pip:
```

Comment:
Please use the `python-` prefix. We generally want to stick with the debian style name for consistency.